### PR TITLE
[Light Up Chinatown] partner thanks

### PR DESCRIPTION
**Pull request by mllee**
Added the logos of partner organizations at the bottom with resizing:
image
image

### CODE REVIEW

**Review by danthaman44 — APPROVED** [review]

File: src/locales/en/translation.json [hunk 1]
```diff
@@ -231,6 +231,7 @@
     "donationDetail4": "1 x 12-inch custom designed lantern by a local artist to take home with you ",
     "donationDetail5": "An invitation to the Winter Solstice Celebration and lighting ceremony in December",
     "goal1":"Permanent Light fixtures and lanterns on Mott St from Canal to Bayard St",
-    "goal2":"Permanent Light fixtures on Bayard St from Mott Street to Bowery St"
+    "goal2":"Permanent Light fixtures on Bayard St from Mott Street to Bowery St",
```
Comment (justintmckibben) [inline src/locales/en/translation.json:234]: nit missing space after `:`

File: src/locales/en/translation.json [hunk 2]
```diff
@@ -231,6 +231,7 @@
...
+    "partnerThanks": "Thank you to our Chinatown Partners who make this possible!"
```
Comment (justintmckibben) [inline src/locales/en/translation.json:235]: be sure also to fill in the chinese version

File: src/components/LightUpChinatown/LightUpChinatownPage.tsx [hunk 3]
```diff
@@ -1,6 +1,12 @@
 import React from 'react';
 import styled from 'styled-components';
 import lanternHeroTop from './images/lantern-hero-top.png';
+import lanternFooter from './images/lantern-footer.png';
+import partners_46 from './images/partners_46.png';
```
Comment (justintmckibben) [inline src/components/LightUpChinatown/LightUpChinatownPage.tsx:5]: why do we have a mix of underscores and dashes? could we do one or the other?

File: src/components/LightUpChinatown/LightUpChinatownPage.tsx [hunk 4]
```diff
@@ -99,10 +105,45 @@ const LightUpChinatownPage = () => {
...
+      <Banner>
+        <PartnerThanksTitle>
+          {t('lightUpChinatown.partnerThanks')}
+        </PartnerThanksTitle>
+        <PartnersLogoContainer>
+          <PartnerLogo src={partners_46} alt="Logo" />
+          <PartnerLogo src={partners_ccba} alt="Logo" />
+          <PartnerLogo src={partners_cccny} alt="Logo" />
+          <PartnerLogo src={partners_prm} alt="Logo" />
+          <PartnerLogo src={partners_udooda} alt="Logo" />
```
Comment (justintmckibben) [inline src/components/LightUpChinatown/LightUpChinatownPage.tsx:117]: a more descriptive alt would be great, for usability reasons (not that our website is super optimized for the blind yet, but baby steps)

**Review by justintmckibben — APPROVED** [review]

File: src/locales/en/translation.json (hunk 2 above)
Comment (mllee) [reply src/locales/en/translation.json:235]: how do we go about requesting translations? My chinese is awful

Comment (danthaman44) [reply src/locales/en/translation.json:235]: Just fill the translation.cn file with the english copy for now, usually translation is done after

**Pull request merged**